easybuild/easyconfigs/f/freeglut/freeglut-3.6.0-GCCcore-13.3.0.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'freeglut'
+version = '3.6.0'
+
+homepage = 'http://freeglut.sourceforge.net/'
+description = "freeglut is a completely OpenSourced alternative to the OpenGL Utility Toolkit (GLUT) library."
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['9c3d4d6516fbfa0280edc93c77698fb7303e443c1aaaf37d269e3288a6c3ea52']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('CMake', '3.29.3'),
+]
+dependencies = [
+    ('X11', '20240607'),
+    ('Mesa', '24.1.3'),
+    ('libGLU', '9.0.3'),
+]
+
+configopts = ' -DX11_X11_LIB="$EBROOTX11/lib/libX11.so"  -DX11_X11_INCLUDE_PATH="$EBROOTX11/include/X11"  -DX11_Xext_LIB="$EBROOTX11/lib/libXext.so"  -DX11_Xrandr_LIB="$EBROOTX11/lib/libXrandr.so"  -DX11_Xrandr_INCLUDE_PATH="$EBROOTX11/include/X11/extensions/"  -DX11_Xi_LIB="$EBROOTX11/lib/libXrandr.so"  -DX11_Xi_INCLUDE_PATH="$EBROOTX11/include/X11/extensions/" '
+
+
+sanity_check_paths = {
+    'files': [('lib/libglut.a', 'lib64/libglut.a'), ('lib/libglut.so', 'lib64/libglut.so')],
```

please use SHLIB_EXT instead of .so
also, it's not needed to check for both lib and lib64: EB automatically creates the symlinks
```suggestion
    'files': ['lib/libglut.a', 'lib/libglut.%s' % SHLIB_EXT],
```